### ISSUE TITLE
 Unable to locate package dotnet-sdk-2.2

### ISSUE THREAD

**Issue by andybarzyk** [dotnet/core#2650]
Problem encountered on https://dotnet.microsoft.com/download/linux-package-manager/ubuntu18-10/sdk-current
Operating System: Linux Ubuntu 18.10 - x64

Provide details about the problem you are experiencing. Include your operating system version, exact error message, code sample, and anything else that is relevant.

**Comment by andybarzyk** [comment]
followed instructions on https://dotnet.microsoft.com/download/linux-package-manager/ubuntu18-10/sdk-current
-----------------------------------------------------
~$ sudo apt-get install dotnet-sdk-2.2
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package dotnet-sdk-2.2
E: Couldn't find any package by glob 'dotnet-sdk-2.2'
E: Couldn't find any package by regex 'dotnet-sdk-2.2'

**Comment by andybarzyk** [comment]
I had it working but wanted to upgrade and accidentally installed 3.0  for ARM 64. the command dotnet would only yield binary error. Tried removing and re-installing .net 2.2 --- this was the error...

**Comment by dagood** [comment]
Can you run this command to show your configured repositories?

```
# Shows the repositories you have configured. Source: https://askubuntu.com/a/741948
grep -r --include '*.list' '^deb ' /etc/apt/sources.list /etc/apt/sources.list.d/
```

Also, have you tried the workaround commands that show up in the 18.04 instructions? https://dotnet.microsoft.com/download/linux-package-manager/ubuntu18-04/sdk-current

(The workaround commands not being in the 18.10 instructions is, IMO, a bug: https://github.com/dotnet/core/issues/2486.)

**Comment by miaodadao** [comment]
@dagood 
```
root@ubuntu:~# grep -r --include '*.list' '^deb ' /etc/apt/sources.list /etc/apt/sources.list.d/
/etc/apt/sources.list:deb http://cn.archive.ubuntu.com/ubuntu/ xenial main restricted
/etc/apt/sources.list:deb http://cn.archive.ubuntu.com/ubuntu/ xenial-updates main restricted
/etc/apt/sources.list:deb http://cn.archive.ubuntu.com/ubuntu/ xenial universe
/etc/apt/sources.list:deb http://cn.archive.ubuntu.com/ubuntu/ xenial-updates universe
/etc/apt/sources.list:deb http://cn.archive.ubuntu.com/ubuntu/ xenial multiverse
/etc/apt/sources.list:deb http://cn.archive.ubuntu.com/ubuntu/ xenial-updates multiverse
/etc/apt/sources.list:deb http://cn.archive.ubuntu.com/ubuntu/ xenial-backports main restricted universe multiverse
/etc/apt/sources.list:deb http://security.ubuntu.com/ubuntu xenial-security main restricted
/etc/apt/sources.list:deb http://security.ubuntu.com/ubuntu xenial-security universe
/etc/apt/sources.list:deb http://security.ubuntu.com/ubuntu xenial-security multiverse
/etc/apt/sources.list.d/microsoft-prod.list:deb [arch=amd64] https://packages.microsoft.com/ubuntu/16.04/prod xenial main
root@ubuntu:~# lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.6 LTS
Release:	16.04
Codename:	xenial
root@ubuntu:~# 
```

**Comment by dagood** [comment]
Closing: no new info provided on the original issue for a week. The workaround instructions are now available on the 18.10 download page, so it's likely future instances of this problem will be mitigated by the workaround being easier to find.

---

@miaodadao if you're experiencing a similar issue, please file an issue including the commands you're running and error you're receiving and answer this question as well:

> Also, have you tried the workaround commands that show up in the 18.04 instructions?

Thanks.
